File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for (int i = 0; i<nums.length; i++){
            int check = 0;
            for (int j = 0; j<nums.length; j++){
                if(nums[i]==nums[j]){
                    check++;
                }
                if(check==2) return true;
            }

            check = 0;
        }
        return false;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> result;
        map <int,int> mp;
        int temp;
        for (int i =0;i<nums.size();i++)
        {
            mp[nums[i]]=i+1;
        }

        for (int i = 0;i<nums.size();i++)
        {
            temp = target-nums[i];
            if(mp[temp]!=0)
            {
                if(i==mp[temp]-1)
                {continue;}
                result.push_back(i);
                result.push_back((mp[temp])-1);
            return result;
            }   
        }
        return result;
    }
};